client/components/plans/plan-actions/index.jsx

```diff
@@ -225,7 +234,9 @@ const PlanActions = React.createClass( {
 			const link = purchasesPaths.managePurchase( this.props.site.slug, this.props.sitePlan.id );
 
 			return (
-				<a href={ link } className="button plan-actions__upgrade-button">{ this.translate( 'Manage Plan', { context: 'Link to current plan from /plans/' } ) }</a>
+				<a href={ link } className="button plan-actions__upgrade-button">{
+						this.translate( 'Manage Plan', { context: 'Link to current plan from /plans/' } )
+				}</a>
```

Comment:
it was about the formatting/style of putting the brackets with the `<a />` tag and not with `this.translate()`